### PR TITLE
Chapter_7

### PR DESCRIPTION
**Pull request by stephanie1997-kek**

### CODE REVIEW

**Comment by stephanie1997-kek** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -26,6 +42,55 @@ textarea {
...
+body {
+  padding-top: 60px;
+}
+
+/* header */
+#logo {
+  float: left;
+  margin-right: 10px;
+  font-size: 1.7em;
+  color: #fff;
```
Comment (saobv-2223) [inline app/assets/stylesheets/custom.scss]: image

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -26,6 +42,55 @@ textarea {
...
+  text-transform: uppercase;
+  letter-spacing: -1px;
+  padding-top: 9px;
+  font-weight: bold;
+  &:hover {
```
Comment (saobv-2223) [inline app/assets/stylesheets/custom.scss]: image

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -26,6 +42,55 @@ textarea {
...
+  }
+
+  small {
+    float: left;
+  }
+
+  ul {
+    float: right;
+    list-style: none;
+  li {
```
Comment (saobv-2223) [inline app/assets/stylesheets/custom.scss]: and it

File: app/assets/stylesheets/custom.scss [hunk 4]
```diff
@@ -45,58 +110,82 @@ h2 {
...
 
-  small {
-    float: left;
-  }
+input.btn.btn-primary{
+  margin-top: 10px;
+}
 
+#error_explanation {
+  color: $red;
```
Comment (saobv-2223) [inline app/assets/stylesheets/custom.scss:181]: color code

File: config/settings.yml [hunk 5]
```diff
@@ -8,3 +8,13 @@ links:
...
+  gravatar: https://secure.gravatar.com/avatar/
+
+validations:
+  name:
+    max_length: 50
+  email:
+    max_length: 225
+    regex: !ruby/regexp '/\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i'
+  password:
+    min_length: 6
```
Comment (saobv-2223) [inline config/settings.yml]: add endline, please

File: app/assets/stylesheets/custom.scss (hunk 2 above)
Comment (stephanie1997-kek) [reply app/assets/stylesheets/custom.scss]: done

File: app/assets/stylesheets/custom.scss (hunk 3 above)
Comment (stephanie1997-kek) [reply app/assets/stylesheets/custom.scss]: done

File: config/settings.yml (hunk 5 above)
Comment (stephanie1997-kek) [reply config/settings.yml]: done

File: app/assets/stylesheets/custom.scss (hunk 4 above)
Comment (stephanie1997-kek) [reply app/assets/stylesheets/custom.scss:181]: done

File: app/assets/stylesheets/custom.scss (hunk 1 above)
Comment (stephanie1997-kek) [reply app/assets/stylesheets/custom.scss]: image
I already have a .scss file for color.

**Comment by stephanie1997-kek** [comment]
ready

File: app/views/users/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<% provide :title, @user.name %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <h1>
+        <%= gravatar_for @user %>
+        <%= @user.name %>
+      </h1>
+    </section>
+  </aside>
```
Comment (saobv-2223) [inline app/views/users/show.html.erb:10]:
```suggestion
  <aside class="col-md-4">
    <section class="user_info">
      <h1>
        <%= gravatar_for @user %>
        <%= @user.name %>
      </h1>
    </section>
  </aside>
</div>
```

Comment (stephanie1997-kek) [reply app/views/users/show.html.erb:10]: done

**Comment by stephanie1997-kek** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
merge

**Pull request merged**